### ISSUE TITLE
Integration with Videojs-resolution-switcher

### ISSUE THREAD

**Issue by adri21112** [yanwsh/videojs-panorama#114]
Hi!

I am working with your fantastic plugin and I have some problems to integrate with Videojs-resolution-switcher. How can I switcher the video texture when I change the video quality with this plugin?

Thanks for your attention!!!

**Comment by yanwsh** [comment]
Yes, I think it should work, just try and let me know if you have any issues.

**Comment by adri21112** [comment]
Yep... I did more test and I can view the changes.

Thanks
